webapp/src/DOMJudgeBundle/Command/ImportEventFeedCommand.php

```diff
@@ -1217,10 +1261,18 @@ protected function importSubmission(array $event)
                         return;
                     }
 
-                    $client   = new Client();
-                    $response = $client->get($zipUrl, ['sink' => $zipFile]);
-                    if ($response->getStatusCode() !== 200) {
-                        $this->logger->error(sprintf('Can not download ZIP for submission %s', $submissionId));
+                    $client = new Client();
+                    try {
+                        $response = $client->get($zipUrl, ['sink' => $zipFile]);
+                        if ($response->getStatusCode() !== 200) {
```

Comment:
Please add a TODO here (which doesn't have to be resolved with this commit) to do make this more reliable with retrying.

Reply:
I will. Note that re-starting the import will retry currently, so there is *some* way to fix it.

Reply:
We might want to use something like https://github.com/caseyamcl/guzzle_retry_middleware to make life easy when we do implement it.